label_inspector/analysis/label_analysis.py

```diff
@@ -189,6 +189,18 @@ def punycode_compatibility(self) -> Optional[str]:
     def punycode_encoding(self) -> Optional[str]:
         return self._punycode_analysis.encoded
 
+    @field
+    def canonical_label(self) -> Optional[str]:
+        canonicals = []
+        for grapheme in self._graphemes_untruncated:
+            if not grapheme._is_confusable:
+                canonicals.append(grapheme.value)
+            elif grapheme.confusables_canonical is None:
+                return None
```

Comment:
not sure about this but we will investigate it later

Reply:
You mean we could return the grapheme itself when the canonical is unknown?

Reply:
Yes but I will investigate it later.